base.yml

```diff
@@ -0,0 +1,5 @@
+base:
+  environment:
```

Comment:
I think most of this is for dev more than prod isn't it?. I wonder if we could use some of the inscrutable yaml syntax to share this just for the dev config e.g:

E.g:

```yaml
service:
  environment: *env
    - PYTHONUNBUFFERED=1
    - PYTHONDONTWRITEBYTECODE=1
    - TERM=xterm-256color
  extends:
    file: ../common.yml
    service: service
  links:
    - solitude:solitude

solitude:
  environment: &env
  extends:
    file: ../common.yml
    service: service
  volumes:
    - ./solitude/:/srv/solitude
  links:
    - mysql:mysql

mysql:
  extends:
    file: ../common.yml
    service: mysql
  volumes_from:
    - mysqldata

mysqldata:
  extends:
    file: ../common.yml
    service: mysqldata


```